src/ios/orb.yml

```diff
@@ -370,6 +374,40 @@ commands:
           command: test-without-building
           arguments: $XCODEBUILD_ARGS -destination "platform=iOS Simulator,id=$SIMULATOR_UDID"
       - save-xcodebuild-artifacts
+  with-cocoapods-specs-cache:
+    parameters:
+      cached-steps:
+        description: Steps that will be specs cache is restored, but before it is saved.
```

Comment:
If I'm reading this right, I think it might be more clear as "Steps that will be **run after the** specs cache is restored, but before it is saved.

Reply:
Done!